src/init.rs

```diff
@@ -132,10 +154,10 @@ starship_precmd() {
     if [[ $STARSHIP_START_TIME ]]; then
         STARSHIP_END_TIME=$(date +%s)
         STARSHIP_DURATION=$((STARSHIP_END_TIME - STARSHIP_START_TIME))
-        PS1="$(starship prompt --status=$STATUS --jobs="$(jobs -p | wc -l)" --cmd-duration=$STARSHIP_DURATION)"
+        PS1="$(STARSHIP_EXEC prompt --status=$STATUS --jobs="$(jobs -p | wc -l)" --cmd-duration=$STARSHIP_DURATION)"
```

Comment:
On first reading, I thought `STARSHIP_EXEC` was a shell variable. It wasn't until second reading that I realize that it's going to be nuked before it ever makes it to the shell.

Could we make this a string that's invalid as a shell variable, e.g. `## STARSHIP ##` or something so that there's no chance someone thinks it's executed as shell code?

Reply:
Good idea. I renamed it to `## STARSHIP ##`.